docker-compose.yaml

```diff
@@ -4,8 +4,9 @@ services:
       context: https://github.com/alex3025/ilo-fans-controller.git#main
       dockerfile: Dockerfile
     ports:
-      - "8084:80"
+      # You can use whatever (unused) port instead of 8000.
+      - "8000:80"
     environment:
-      ILO_HOST: 192.168.1.2
-      ILO_USERNAME: Administrator
-      ILO_PASSWORD: MyPa$$word
+      ILO_HOST: 'your-ilo-address'  # Ex. 192.168.1.69
```

Comment:
I specifically left the real values in here to show these quotes aren't needed.

Reply:
Oh ok, anyway I don't think they will hurt someone.